### PR TITLE
Update docs for chmod laminate and bootstrap.sh

### PR DESCRIPTION
**Pull request by tonyhutter**
### Description
- Update docs to talk about `chmod()` based lamination.
- Minor update to` bootstrap.sh` docs.

### Motivation and Context
Update the docs

### How Has This Been Tested?
Locally built and verified HTML pages

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (a change to man pages or other documentation)

### Checklist:

- [ ] My code follows the UnifyFS code style requirements.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commit messages are properly formatted.

### CODE REVIEW

File: docs/assumptions.rst [hunk 1]
```diff
@@ -28,22 +28,27 @@ Application Behavior
...
-In the first version of UnifyFS, eventually, a process declares the file to be
-laminated through a UnifyFS API call.
-After a file has been laminated, the contents of the file cannot be changed.
-The file becomes permanently read-only.
-After lamination, any process may freely read any part of the file.
-If the application process group fails before a file has been laminated,
-UnifyFS may delete the file.
-An application can delete a laminated file.
+One key aspect of UnifyFS is the idea of "laminating" a file.  After a file is
+laminated, it's becomes "set in stone" and its data is accessible across all the
```
Comment (adammoody) [inline docs/assumptions.rst]: edit: it's becomes --> it becomes

File: docs/assumptions.rst [hunk 2]
```diff
@@ -28,22 +28,27 @@ Application Behavior
...
-operations. As part of the UnifyFS project, we will investigate these implicit
-lamination conditions to determine the best way to enable lamination of files
-without explicit UnifyFS API calls being made by the application.
-
-In the first version of UnifyFS, eventually, a process declares the file to be
-laminated through a UnifyFS API call.
-After a file has been laminated, the contents of the file cannot be changed.
-The file becomes permanently read-only.
-After lamination, any process may freely read any part of the file.
-If the application process group fails before a file has been laminated,
```
Comment (adammoody) [inline docs/assumptions.rst]: Some of the lines in this paragraph probably still apply, like letting Unify delete a file if a process group fails before laminating a file.

File: docs/assumptions.rst [hunk 3]
```diff
@@ -28,22 +28,27 @@ Application Behavior
...
-UnifyFS may delete the file.
-An application can delete a laminated file.
+One key aspect of UnifyFS is the idea of "laminating" a file.  After a file is
+laminated, it's becomes "set in stone" and its data is accessible across all the
+nodes.  Laminated files are read-only, and cannot be modified in any way (but
+can be deleted).
+
+The typical use case is to laminate your checkpoint files after they've been
+written.  To laminate a file, simply call chmod() to remove all the write bits
+on the file.  If you have written to the file, you will also need to sync
```
Comment (adammoody) [inline docs/assumptions.rst]: Let's write this part to be process-centric so it's clear that any process that writes to a file must also fsync, like "If a process has written to the file, it must also sync ..."

**Comment by tonyhutter** [comment]
@adammoody I pushed out some changes to address your comments.  Can you please take another look?

**Comment by adammoody** [comment]
Thanks, @tonyhutter

**Pull request merged**